go/Makefile

```diff
@@ -86,7 +101,17 @@ build-loki: loki-api
 loki-api:
 	go build -o loki-api ./cmd/loki/main.go
 
+.PHONY: build-ddapi
+build-ddapi: ddapi
+
+ddapi:
+	go build -o ddapi ./cmd/ddapi/ddapi.go
+
 
 .PHONY: unit-tests
 unit-tests:
+	# Let's see if the Docker daemon of the bk host is available
+	echo -e "GET /images/json HTTP/1.0\r\n" | nc -U /var/run/docker.sock
```

Comment:
Instead of adding an external dependency try using [docker info](https://docs.docker.com/config/daemon/#check-whether-docker-is-running)

Reply:
Oh, you're already looking! :)

Yeah, this is just a work in progress debugging commit, for debugging why the go docker client library throws

```
failed to create container: creating reaper failed: could not start container: dial tcp: i/o timeout
```

Note that this is executed in a container and I thought that in that container we do not have the `docker` command available. That's why I opted for going straight to the socket :) 

Now I see that I've been there before lol, and this is exactly why we added this to the CI container Dockerfile:
```
# Set up the Docker binaries so that within the container we can manage
# containers running on the host with the `docker` CLI (the host's Docker
# socket is going to be mounted in).
RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-19.03.5.tgz && \
    tar --strip-components=1 -xvzf docker-19.03.5.tgz -C /usr/local/bin && \
    rm -f docker-19.03.5.tgz
```